### ISSUE TITLE
Unreachable concepts uncovered by configlet lint

### ISSUE THREAD

**Issue by jmrunkle** [exercism/java#1992]
```
The Concept Exercise `elons-toy-car` has `conditionals` in its `prerequisites`, which is not in the `concepts` array of any other user-facing Concept Exercise:
/home/runner/work/java/java/config.json

The Concept Exercise `remote-control-competition` has `lists` in its `prerequisites`, which is not in the `concepts` array of any other user-facing Concept Exercise:
/home/runner/work/java/java/config.json

The Concept Exercise `football-match-reports` has `variables` in its `prerequisites`, which is not in the `concepts` array of any other user-facing Concept Exercise:
/home/runner/work/java/java/config.json

The Concept Exercise `football-match-reports` has `methods` in its `prerequisites`, which is not in the `concepts` array of any other user-facing Concept Exercise:
/home/runner/work/java/java/config.json
```

Basically, these are unreachable so we are going to mark them wip until we can implement the other concepts that make it so that we cannot reach these.

**Comment by jmrunkle** [comment]
Apparently the issue is even more extensive because `elons-toy-car` taught the `classes` concept which blocks access to `need-for-speed` and `wizards-and-warriors`. Let us see how deep this issue goes...

**Comment by jmrunkle** [comment]
That appears to be the end of the chain of unreachable concepts.

Here is what we need to be able to re-enable everything:

* `conditionals` - unblocks `elons-toy-car` which in turn unblocks `need-for-speed` and `wizards-and-warriors`
* `lists` - unblocks `remote-control-competition`
* `variables` and `methods` - unblocks `football-match-reports`

I will create one issue per missing concept and keep this one open waiting for those to be fixed.

**Comment by jmrunkle** [comment]
Looks like `lists` already has an issue: https://github.com/exercism/java/issues/1862

**Comment by jmrunkle** [comment]
Added https://github.com/exercism/java/issues/1994 for `conditionals` with a lot of TODOs on that first comment...

**Comment by jmrunkle** [comment]
OK, as near as I can tell, the `variables` and `methods` concepts should be taught in the basic `lasagna` exercise: https://github.com/exercism/java/blob/main/exercises/concept/lasagna/.docs/introduction.md

If possible, I think we can either (1) remove those concepts, or (2) mark them as included in the `lasagna` exercise.

**Comment by jmrunkle** [comment]
OK, I looked at what is described in the `basics` exercise and it clearly includes `variables` and `methods` so I am just going to remove those two concepts from config.json.

**Comment by jmrunkle** [comment]
That fixes part of the problem (the pre-reqs), but we still need to fix another issue before we can re-enable `football-match-reports`: we never added a slug to config.json for the `switch-statement` concept.
